paper-server/src/main/java/org/bukkit/craftbukkit/entity/CraftCreaking.java

```diff
@@ -14,6 +20,34 @@ public Creaking getHandle() {
         return (Creaking) this.entity;
     }
 
+    @Override
+    public Location getHome() {
+        return CraftLocation.toBukkit(this.getHandle().getHomePos(), this.getHandle().level());
```

Comment:
This seems like it would fail inside the toBukkit method if getHomePos returns null